### PR TITLE
Added entity panel toggle setting as a checkbox

### LINKED ISSUE #270: Option to hide other players' entity name panels on Momentum Settings panel
Currently there is no way to hide other players' playermodels and trails. This can be pretty distracting. 

This would require an option in the Momentum settings panel, probably just a checkbox, to "hide other players". 

Suggested by several

### PR DESCRIPTION
**Pull request by braem**
Closes #270 

Added `ToggleEntityPanels` to `resource/ui/SettingsPanel_OnlineSettings.res` under "Enable online ghost flashlight sounds". This setting toggles `mom_entpanels_enable`.
Added the text refs for this in `resource/momentum_english_ref.res` and ran the tokenizer.

image

### Checklist
- [x] If there is a localization token change, I have updated the `momentum_english_ref.res` file with the changes, ran `tokenizer.py` to generate an up-to-date localization file, and have committed both the `.res` file changes and the new localization `.txt` file
- [x] If I introduced new h/cpp files, I have added them to the appropriate project's VPC file (`server_momentum.vpc` / `client_momentum.vpc` / etc)
- [x] My commits are relatively small and scoped to the best of my ability
- [x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
- [x] My branch is functionally complete; the only changes to be done will be those potentially requested in code review

### CODE REVIEW

**Comment by braem** [comment]
Done :)

File: mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res [hunk 1]
```diff
@@ -79,13 +79,44 @@
 		"Font"		"DefaultVerySmall"
 	}
     
+	"ToggleEntityPanels"
```
Comment (Gocnak) [inline mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res]: This also needs to be declared in the code. See how the other elements are declared in the OnlineSettings class.

Comment (braem) [reply mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res]: In doing this I found that the only checkbox declared in the code (`HudSettingsPage`) is the one for `mom_ghost_online_alpha_override_enable`. Should I add the others as well?

Comment (Gocnak) [reply mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res]: Yes.

Comment (braem) [reply mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res]: Done.

File: mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res [hunk 2]
```diff
@@ -79,13 +79,44 @@
...
+		"wrap"		"0"
+		"centerwrap"		"0"
+		"textinsetx"		"6"
+		"textinsety"		"0"
+		"auto_wide_tocontents"		"1"
+        "auto_tall_tocontents" "1"
+		"use_proportional_insets"		"0"
+		"Default"		"0"
+		"cvar_name"		"mom_entpanels_enable"
+		"cvar_value"		"1"
```
Comment (Gocnak) [inline mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res]: Alright, last but not least, remove all of the `cvar_value` keys from this file. This file is old and the previous way to define cvars was through the res file, but defining `cvar_value` makes the game not "save" your settings if the cvar is archived.

Comment (braem) [reply mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res]: Ah yeah I see that it is done when making cvars in code. Removed them as requested.

**Pull request merged**